CalcualtorLogic.js

```diff
@@ -99,6 +99,10 @@ class Calculator {
         const currentResultAndclickedButton = this.currentResult + event.target.dataset.number;
         this.currentResult = currentResultAndclickedButton;
     }
+    updateResultElement() {
+
```

Comment:
this method is empty.
it's not doing what it should be doing?!!!!

this method should get the current value of the `currentResult` property and assign this value to the `resultElement`

Remember `resultElement` is an object that represent HTML element 
how do you update the text of a HTML element?